pra_request_tracker/record_requests/models.py

```diff
@@ -0,0 +1,24 @@
+from django.db import models
+
+from pra_request_tracker.users.models import User
+
+
+class RecordRequest(models.Model):
+    requester = models.ForeignKey(User, on_delete=models.SET_NULL, null=True)
+    description = models.TextField()
+    agency = models.CharField(max_length=100)
+
+    def __str__(self):
+        return f"RecordRequest({self.id}) to {self.agency}"
+
+
+class RecordRequestFile(models.Model):
+    request = models.ForeignKey(
+        RecordRequest,
+        on_delete=models.CASCADE,
+    )
+    url = models.URLField()
```

Comment:
A lot of the time, various agencies will just email back a CSV/PDF/zip file with the data. I think we might need this to be a `FileField` instead of a `URLField` so users can upload files directly

Reply:
Oh yeah we can just have blobs directly in the DB, duh.

Reply:
I think the way `FileField` works, depending on which storage driver you use (we'll probably use the default) - it saves the path that the file gets written out to in the DB and then will load from that path should the file need to be accessed ([docs](https://docs.djangoproject.com/en/3.2/ref/models/fields/#filefield) I'm basing this off of and hoping I'm understanding lol). Blobs deffo can be stored in the database but I think for most cases it's not ideal.

Reply:
After some further discussion offline, we're going to stick with URL! The objects can just be shipped to S3/Spaces